modules/oe_content_department/config/install/field.storage.rdf_entity.oe_department_name.yml

```diff
@@ -0,0 +1,25 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - rdf_entity
+    - taxonomy
+third_party_settings:
+  rdf_entity:
+    mapping:
+      target_id:
+        predicate: 'http://europa.eu/department-name-reference'
```

Comment:
Shouldn't we add the proper mapping to this field and leave the title without a mapping?

Reply:
@imanoleguskiza No because the title of department according to the schema is a text type of data, not a resource (reference) to something else that is another construct. Our solution is just a workaround to the need to limit the title to existing departments. 

But I did update this mapping to make it a resource :)